### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by FrancisJMR**
New Researchers ID & Access Claims landing page.  flow_of_claims.png still needs to be added.

### CODE REVIEW

File: researcher_ids/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Research groups around the world are making datasets available to other researchers through data sharing repositories such as EGA and dbGaP. However, access processes are fraught with difficulties and the following shortcomings have been identified:
+
+- It is inconsistent: different Data Access Committees (DACs) may reach different conclusions regarding appropriate access.
+- It is cumbersome:  DACs have a difficult time identifying who a bona fide researcher is, and have difficulty validating their identity.
+- It is not scalable: the number of requests for access to human subjects data is growing at a supralinear rate, and it is growing increasingly difficult for DACs to keep up.
+
+The genomics community was therefore in need of:
+
+- a consistent definition of who a bona fide researcher is
+- ne or more identity providers that respect this definition and provide identities in the virtual world that travel with the researcher across various data sharing repositories
```
Comment (cdvoisin) [inline researcher_ids/README.md]: Replace "ne" with "one"?

File: researcher_ids/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    2. Other Claim Clearinghouses will read the bytes from another service using special access token that it holds which contains permission to do so, then return the bytes back to the researcher.
+    3. Yet other varieties of Claim Clearinghouses will just proceed taking an action internally or reading bytes directly from a database in and of itself without much need for downstream access tokens.
+5. Some systems will use the Passport once, exchange it for a downstream access token, and not use the Passport again. Other systems will use the Passport multiple times until the passport token expires.
+    1. Some systems can issue refresh tokens that can be exchanged for new Passport tokens with a fresh set of claims and updated expiry timestamp.
+    2. Other systems do not issue refresh tokens, and the only option to get a new Passport is to have the user do a fresh login (OIDC Authorization Flow) to fetch a new Passport.
+
+## Flow Of Claims
+
+![alt text](https://github.com/ga4gh-duri/flow_of_claims.png "Flow of Claims")
```
Comment (cdvoisin) [inline researcher_ids/README.md]: The PNG file is missing?

**Review by cdvoisin — COMMENTED** [review]
Thanks for putting this together. How about we land it with the current links and then update the links later to point to the github docs instead of the Google docs.

File: researcher_ids/README.md (hunk 2 above)
Comment (cdvoisin) [reply researcher_ids/README.md]: Also it may be worth putting image assets in the /assets/img/ directory.

File: researcher_ids/README.md (hunk 1 above)
Comment (FrancisJMR) [reply researcher_ids/README.md]: Fixed in new commit.

File: researcher_ids/README.md (hunk 2 above)
Comment (FrancisJMR) [reply researcher_ids/README.md]: I've changed the path to /assets/img/  --- I'd don't have push access to the repository to add the image in that folder however. Can you try it? The image should be named "flow_of_claims.png"

**Review by cdvoisin — APPROVED** [review]

**Comment by FrancisJMR** [comment]
Ok I've added the png file and incorporated the changes to the introduction from Tommi and I found in the Google Doc.  I cannot merge this back to master but it should be good to go.

File: researcher_ids/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Research groups around the world are making datasets available to other researchers through data sharing repositories such as EGA and dbGaP. However, access processes are fraught with difficulties and the following shortcomings have been identified:
+
+- It is inconsistent: different Data Access Committees (DACs) may reach different conclusions regarding appropriate access.
+- It is cumbersome:  DACs have a difficult time identifying who a bona fide researcher is, and have difficulty validating their identity.
+- It is not scalable: the number of requests for access to human subjects data is growing at a supralinear rate, and it is growing increasingly difficult for DACs to keep up.
+
+The genomics community was therefore in need of:
+
+- a consistent definition of who a bona fide researcher is
+- one or more identity providers that respect this definition and provide secure digital identities that researchers can use online to access in the virtual world that travel with the researcher across various federated data sharing repositories around the world
```
Comment (cdvoisin) [inline researcher_ids/README.md]: Remove: "in the virtual world that travel with the researcher across"
Remove: "sharing" (after the words "federated data")

**Pull request merged**